### PR TITLE
track koyo finance USD stableswap lp token price

### PR DESCRIPTION
**Pull request by perpetuum7**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/koyo exports TVL: 
        
 
 
    boba-treasury             5.76 k
    boba                      5.61 k
    treasury                  5.76 k
    
    total                    5.61 k

**Comment by llamatester** [comment]
The adapter at projects/shibui exports TVL: 
        
 
 
    boba                      0
    boba-pool2                10.79 k
    boba-staking              53.637375
    boba-treasury             25.71 k
    pool2                     10.79 k
    staking                   53.637375
    treasury                  25.71 k
    
    total                    0

File: projects/koyo/index.js [hunk 1]
```diff
@@ -80,6 +84,15 @@ const chainTreasury = (chain) => {
       transform
     );
 
+    await sumKoyoLPTokens(
```
Comment (g1nt0ki) [inline projects/koyo/index.js:87]: sorry, isnt this double counting? these assets are already counted as tvl

Comment (perpetuum7) [reply projects/koyo/index.js:87]: I can see it on the our side but not really on any other protocols side. The way I saw it is the same way as other protocols holding Uniswap LP tokens.
Do you consider it double-counting on our (Koyo) side or it being included in other protocols?

Comment (g1nt0ki) [reply projects/koyo/index.js:87]: shibui treasury is fine, but not your own

Comment (perpetuum7) [reply projects/koyo/index.js]: > shibui treasury is fine, but not your own

Fixed.

**Comment by llamatester** [comment]
The adapter at projects/koyo exports TVL: 
        
 
 
    boba                      5.63 k
    boba-treasury             4.62 k
    treasury                  4.62 k
    
    total                    5.63 k

**Comment by llamatester** [comment]
The adapter at projects/shibui exports TVL: 
        
 
 
    boba                      0
    boba-staking              53.546899999999994
    boba-pool2                10.81 k
    boba-treasury             25.76 k
    staking                   53.546899999999994
    pool2                     10.81 k
    treasury                  25.76 k
    
    total                    0

**Pull request merged**